tests/phpunit/tests/rest-api/rest-revisions-controller.php

```diff
@@ -234,6 +256,28 @@ public function test_get_item_invalid_parent_post_type() {
 		$this->assertErrorResponse( 'rest_post_invalid_parent', $response, 404 );
 	}
 
+	/**
```

Comment:
@ramonjd I would like these invalid tests to added to `wpRestTemplateRevisionsController.php`. I like testing unhappy paths as well as happy paths.

Reply:
Good call, thanks.

Done in ae49d86c2070ac1bc06461a52f52c1d5a42301bb